integration/test_sync_apis.js

```diff
@@ -26,7 +26,11 @@ module.exports = {
         ackWorkerInterval: 100, 
         schedulerInterval: 100, 
         schedulerLockName: 'test:syncApi:lock', 
-        useCache: true});
+        useCache: true,
+        cuidGenerator: function(params) {
```

Comment:
but `selector` doesn't really right either I think. The user can `generate` some hash values from some fields, if that's way they prefer. But I think they are doing the same thing. 
Perhaps name it to `cuidProducer`? This way we won't imply how the cuid will be produced.